lib/sqlalchemy/orm/query.py

```diff
@@ -648,6 +648,18 @@ def only_return_tuples(self, value):
         """
         self._only_return_tuples = value
 
+    @property
+    def is_single_entity(self):
+        """Returns True if this query returns a single entity for each instance in its
```

Comment:
add a single summary line at the top:

```
"indicate if this :class:`.Query` returns tuples or single entities"
```
then a newline, then the paragraph you have here.